pulser/_seq_drawer.py

```diff
@@ -204,6 +204,12 @@ def phase_str(phi):
 
         if draw_phase_area:
             top = False  # Variable to track position of box, top or center.
+            draw_phase = True
+            if all(
+                seq_.type.phase == 0 for seq_ in seq._schedule[ch]
+                if isinstance(seq_.type, Pulse)
+            ):
+                draw_phase = False
```

Comment:
This is perfectly fine, but I guess you can be a bit more economical:
```suggestion
            draw_phase = any(
                seq_.type.phase != 0 for seq_ in seq._schedule[ch]
                if isinstance(seq_.type, Pulse)
            )
```
What do you think?

Reply:
Thanks! I changed the condition..